src/dsql/parse.y

```diff
@@ -2707,9 +2707,13 @@ procedure_clause
 	| external_procedure_clause
 	;
 
+%type <createAlterProcedureNode> change_opt_procedure_clause
+change_opt_procedure_clause
```

Comment:
What is this empty rule?

Reply:
It's not empty, the definition is below.

Reply:
And why you "declared it" here?
```
%type <createAlterProcedureNode> change_opt_procedure_clause
change_opt_procedure_clause
	;
```